### PR TITLE
chore(arn): include ARN of AWS accounts

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Include ARN of AWS account.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/config/config_recorder_all_regions_enabled/config_recorder_all_regions_enabled.py [hunk 1]
```diff
@@ -8,7 +8,10 @@ def execute(self):
         for recorder in config_client.recorders:
             report = Check_Report_AWS(self.metadata())
             report.region = recorder.region
-            report.resource_id = "" if not recorder.name else recorder.name
+            report.resource_arn = config_client.audited_account_arn
```
Comment (jfagoagas) [inline prowler/providers/aws/services/config/config_recorder_all_regions_enabled/config_recorder_all_regions_enabled.py]: Can you do the same as in the `resource_id` here?

File: prowler/providers/aws/services/iam/iam_service.py [hunk 2]
```diff
@@ -54,6 +54,7 @@ def __init__(self, audit_info):
         self.account = audit_info.audited_account
         self.audit_resources = audit_info.audit_resources
         self.partition = audit_info.audited_partition
+        self.account_arn = f"arn:{self.partition}:iam::{self.account}:root"
```
Comment (jfagoagas) [inline prowler/providers/aws/services/iam/iam_service.py]: Why you are not using `audited_account_arn` here?

File: prowler/providers/common/audit_info.py [hunk 3]
```diff
@@ -214,6 +215,7 @@ def set_aws_audit_info(self, arguments) -> AWS_Audit_Info:
             current_audit_info.audit_session = assumed_session
             current_audit_info.audited_account = role_arn_parsed.account_id
             current_audit_info.audited_partition = role_arn_parsed.partition
+            current_audit_info.audited_account_arn = f"arn:{current_audit_info.audited_partition}:iam::{current_audit_info.audited_account}:root"
```
Comment (jfagoagas) [inline prowler/providers/common/audit_info.py:218]: I think this value should be filled with the assumed role values, don't?

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Awesome work!!!

Please check my comments <strike>and the failing tests</strike>  to merge it. 🚀

File: prowler/providers/common/audit_info.py (hunk 3 above)
Comment (MrCloudSec) [reply prowler/providers/common/audit_info.py:218]: The current_audit_info variables are already changed with the assumed session information when I use them.

File: prowler/providers/aws/services/config/config_recorder_all_regions_enabled/config_recorder_all_regions_enabled.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/config/config_recorder_all_regions_enabled/config_recorder_all_regions_enabled.py]: The Config Recorders do not have ARNs that is why we should leave the Account ARN.

File: prowler/providers/aws/services/iam/iam_service.py (hunk 2 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/iam/iam_service.py]: Fixed

**Review by jfagoagas — DISMISSED** [review]
🚀

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**